src/store/ChartStore.ts

```diff
@@ -1051,10 +1048,10 @@ class ChartStore {
         }
     }
     calculateYaxisWidth = (price: number) => {
-        if (!price) return;
+        if (!price || this.isBarrierDragging) return;
 
```

Comment:
this flag prevent calculating the width of the yAxis while dragging. Because it can became too wide because of deleting pip size.